src/lib.rs

```diff
@@ -16,12 +16,45 @@ use tracing_subscriber::{
     registry::LookupSpan,
 };
 
+const LINE_VERT: &str = "│";
+const LINE_HORIZ: &str = "─";
+const LINE_HORIZ_THICK: &str = "━";
+const LINE_BRANCH: &str = "┝";
+
+#[derive(Debug)]
+struct Config {
+    pub ansi: bool,
+    pub indent_lines: bool,
+    pub indent_amount: usize,
```

Comment:
Since this type itself is not public, these fields do not need to be public (and making them public doesn't actually do anything).
```suggestion
    ansi: bool,
    indent_lines: bool,
    indent_amount: usize,
```

Reply:
Whoops, good catch!